Changelog.md

```diff
@@ -134,6 +134,8 @@ diaspora.yml file**. The existing settings from 0.4.x and before will not work a
 * Fix missing translation on privacy settings page [#5671](https://github.com/diaspora/diaspora/pull/5671)
 * Fix code overflow for the mobile website [#5675](https://github.com/diaspora/diaspora/pull/5675)
 * Strip Unicode format characters prior post processing [#5680](https://github.com/diaspora/diaspora/pull/5680)
+* Commenting (+ liking) on a post of someone who ignores you to show the reason
+[#5366](https://github.com/diaspora/diaspora/pull/5366)
```

Comment:
I think you can write "Fix ... ignores you **doesn't** show the reason" and put the link on the same line.